src/client/player.ts

```diff
@@ -484,10 +535,44 @@ class UI {
       ]}),
     ]});
     this.elements.sidebarMenu.appendChild(tileInfo);
+    
+    if (tile.improvement.errand) {
```

Comment:
Honestly the whole frontend is a cheap mess... I suppose we haven't focused on fixing it because the end goal is to have multiple clients. But I agree, _eventually_ we should make a much better frontend client, maybe in React?